### PR TITLE
add the common az tag for azure, gcp

### PR DESCRIPTION
**Pull request by technovangelist**

### CODE REVIEW

File: content/guides/tagging.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The following integration sources create tags automatically in Datadog:
 | Amazon Web Services | Region, Account, Hostname,  Org ID,  AZ, Kernel, Image, Instance Type, Security Group,   Tags assigned in console or via AWS tools |
 | AWS SQS | Queue Name |
 | Apache | Apache Host and Port |
-| Azure | Tenant Name, Status, Tags, Subscription ID and Name |
+| Azure | Tenant Name, Status, Tags, Subscription ID and Name, Availability Zone in common with AWS tag after contacting Datadog support |
```
Comment (irabinovitch) [inline content/guides/tagging.md]: Azure, Google, and AWS seemed to use different names for these. I believe its

Availability-zone, zone, and availability_zone respectively.

We should confirm. There was a card in the eclair backlog to unify them.  But not sure if its complete.

Comment (technovangelist) [inline content/guides/tagging.md]: according to the aug 3 release notes, its done

Comment (technovangelist) [inline content/guides/tagging.md]: @irabinovitch is this supposed to be live now?

Comment (irabinovitch) [inline content/guides/tagging.md]: @JohnLZeller to confirm, but i believe it's now fully rolled out.

Comment (technovangelist) [inline content/guides/tagging.md]: @JohnLZeller just confirmed to me that its live

**Pull request merged**